application/kitura/README.md

```diff
@@ -0,0 +1,33 @@
+## Prerequisites
```

Comment:
Before the prerequisites there should be a short description of what the app does, like the one in the django example:
https://github.com/secrethub/examples/tree/master/application/django

And we should also put a banner image before that. I think we can ask @florisvdg to create the banner image.

Reply:
The description will be added. As soon as @florisvdg will be back in the office, I will inquire for banner images for all the examples/guides.